packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx

```diff
@@ -131,12 +146,63 @@ export const WebPropertyDetailPage = (): JSX.Element => {
               aria-label="SPA listing"
             >
               <div className="pf-u-w-33 pf-u-mb-lg pf-u-mt-md">
-                <SearchInput
-                  placeholder="Search by name"
-                  value={searchTerm}
-                  onChange={(value) => setSearchTerm(value?.toLowerCase())}
-                  onClear={() => setSearchTerm('')}
-                />
+                <Split hasGutter className="pf-u-mb-md">
+                  <SplitItem className="pf-u-w-33">
+                    <SearchInput
+                      placeholder="Search by name"
+                      value={searchTerm}
+                      onChange={(value) => setSearchTerm(value?.toLowerCase())}
+                      onClear={() => setSearchTerm('')}
+                    />
+                  </SplitItem>
+                  <SplitItem isFilled />
+                  <SplitItem isFilled />
+                  <SplitItem>
+                    <Select
+                      variant={SelectVariant.single}
+                      aria-label="filter Input"
+                      value="Select Enviroment"
+                      onToggle={setIsFilterOpen.toggle}
+                      onSelect={(e, value) => {
+                        if (value === 'Select Enviroment') {
+                          setFilterByEnv('' as string);
+                        } else {
+                          setFilterByEnv(value as string);
+                        }
+                        setIsFilterOpen.off();
+                      }}
+                      // selections={filtervalue} //To be kept as Select
+                      isOpen={isFilterOpen}
+                      aria-labelledby="filter"
+                    >
+                      {webPropertiesKeys.map((envName) => (
+                        <SelectOption key={uuid()} value={envName} />
```

Comment:
may i know the purpose of this UUID generation. 

For unique key may be use a combo of index + env name. That will be unique. And kindly remove this package if not used

Reply:
And i mean before the map function.

Apply 

```js
<Select value="Select Environment" />
{map-it}
```

Reply:
Also itt should be 2._.8 right. Why .9

Reply:
> may i know the purpose of this UUID generation.
> 
> For unique key may be use a combo of index + env name. That will be unique. And kindly remove this package if not used

Its not a good practice to use index in the key according to eslint

Reply:
Yes it's not. But that's when you use just index only. That's why suggested env-name + index combo. So even if env name ends up duplicate index makes it unique.

It's not good to generrate a uuid on each render.